homework/style.css

```diff
@@ -1,3 +1,163 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+}
+
+html {
+  display: block;
+  color: -internal-root-color;
+}
+body {
+  display: block;
+  margin: 8px;
+  width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #f8f8f8;
+  font-family: 'Roboto', sans-serif;
+  color: rgb(0, 0, 0, 87%);
+  margin-top: 0;
+}
+
+.header {
+  color: white;
+  background-color: #3f51b5;
+  padding: 8px 16px;
+  margin-bottom: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+#container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+table {
+  table-layout: fixed;
+  color: rgb(0, 0, 0, 81%);
+}
+
+td {
+  vertical-align: top;
+}
+
+td.label {
+  font-weight: bold;
+}
+
+td:first-child {
+  width: 100px;
+  min-width: 100px;
+  max-width: 100px;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+}
+
+.repository-selector {
+  margin-left: 16px;
+  font-size: 14px;
+  width: 250px;
+  height: 32px;
+  padding: 2px;
+}
+
+.white_Frame {
+  width: 768px;
+  margin-bottom: 8px;
+  border: none;
+  border-radius: 2px;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.2), 0 3px 4px 0 rgba(0, 0, 0, 0.14),
+    0 3px 3px -2px rgba(0, 0, 0, 0.12);
+}
+
+.left-div {
+  padding: 16px;
+  margin-right: 16px;
+}
+
+.left-div,
+.right-div {
+  flex: 1;
+}
+
+.contributor-header {
+  font-size: 0.8rem;
+  color: rgb(0, 0, 0, 54%);
+  padding: 8px 16px 4px 16px;
+}
+
+.contributor-item {
+  border-bottom: solid 1px rgb(0, 0, 0, 12%);
+  padding: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+}
+
+.contributor-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.contributor-avatar {
+  border-radius: 3px;
+  margin-right: 16px;
+}
+
+.contributor-data {
+  flex: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.contributor-badge {
+  font-size: 12px;
+  padding: 2px 8px;
+  line-height: 1rem;
+  background-color: gray;
+  color: white;
+  border-radius: 4px;
+}
+
```

Comment:
One thing I have noticed is that if you select `AngularJS` option, and then set the mobile/tablet emulator on, select `iphone 5/SE`, the Description field does not `wrap` but `crops` the value, so you can't see the full description.

This should be fixed.